resources/aws/elb.go

```diff
@@ -225,7 +225,7 @@ func (lb ELB) findExisting() (*elb.LoadBalancerDescription, error) {
 	descriptions := resp.LoadBalancerDescriptions
 
 	if len(descriptions) == 0 {
-		return nil, NamedResourceNotFoundError{Name: lb.Name}
+		return nil, NotFoundError{ResourceType: ELBType, Name: lb.Name}
```

Comment:
I would go with `MaskAnyf` for consistency, since that's what we [agreed upon](This is a remnant from before we [agreed on errors](https://github.com/giantswarm/aws-operator/issues/112), so I agree with Tim that we should convert this error too to using `MaskAnyf`. ).

We could open another discussion about the benefits of using an error struct, but I doubt it's worth it at this point.